src/tasks/start/start-backend.ts

```diff
@@ -12,11 +12,13 @@ import { setAllPermissionsOpenly } from './backend/set-permissions'
 import { startGanache } from './backend/start-ganache'
 import { createApp } from './backend/create-app'
 import { updateApp } from './backend/update-app'
+import onExit from '~/src/utils/onExit'
```

Comment:
Introducing a mechanism to close all watchers when the program exits, for any reason.